qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -124,3 +121,12 @@
                     test_dir = '/home'
                     source_file = 'cpuid.src.tar.gz'
                     check_cpuid = './cpuid -r | grep eax=0x40000001 | uniq -c'
+                - memory_protection_key:
+                    flags = "pku ospke"
+                    variants:
+                        - @default_model:
+                            cpu_model_flags += ",+pku"
+                        - host_model:
+                            only HostCpuVendor.amd
+                            auto_cpu_model = no
+                            cpu_model = host
```

Comment:
Seems it's supported since 8.4, you need add the restriction.

Reply:
> Seems it's supported since 8.4, you need add the restriction.

Seems it's supported since 8.3 and I update the code, please help to review, thanks.